src/spikeinterface/core/base.py

```diff
@@ -797,7 +798,11 @@ def load(file_path: Union[str, Path], base_folder: Optional[Union[Path, str, boo
             return extractor
 
         else:
-            raise ValueError("spikeinterface.Base.load() file_path must be an existing folder or file")
+            error_msg = (
+                f"{file_path} is not a file or a folder. It should point out to either a json, pickle file or a "
```

Comment:
```suggestion
                f"{file_path} is not a file or a folder. It should point to either a json, pickle file or a "
```

Tiny grammar improvement, then all good

Reply:
Thanks, phrasal verbs are difficult for me in English.